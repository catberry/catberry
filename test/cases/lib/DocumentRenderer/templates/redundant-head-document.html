<!DOCTYPE html>
<html>
	<head><title>Hello</title></head>
	<body>
		document â€“ %%value%%
		<cat-comp id="1"></cat-comp>
		<cat-async-comp id="2"/>
		<head></head>
		<document></document>
	</body>
</html>
