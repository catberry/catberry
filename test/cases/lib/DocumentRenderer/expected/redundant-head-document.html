<!DOCTYPE html>
<html>
	<head><title>Hello</title></head>
	<body>
		document â€“ document
		<cat-comp></cat-comp>
		<cat-async-comp/>
		<head></head>
		<document></document>
	</body>
</html>
